### PR TITLE
📝 Recommend to use matching versions

### PR DESCRIPTION
**Pull request by bcaudan**
## Motivation

Following #611, improve documentation.

## Changes

- add warning close to xhr shared data structure
- add note on npm setup to recommend using matching versions

---

I have gone over the [contributing](https://github.com/DataDog/browser-sdk/blob/master/CONTRIBUTING.md) documentation.

### CODE REVIEW

**Review by BenoitZugmeyer — APPROVED** [review]

File: packages/logs/README.md [hunk 1]
```diff
@@ -13,19 +13,19 @@ With the `datadog-logs` library, you can send logs directly to Datadog from JS c
 
 ## Setup
 
-**Datadog client token**: For security reasons, [API keys][1] cannot be used to configure the `datadog-logs` library, because they would be exposed client-side in the JavaScript code. To collect logs from web browsers, a [client token][2] must be used. See the [client token documentation][2] for more details.
+**Datadog client token**: For security reasons, [API keys][1] cannot be used to configure the browser logs SDK, because they would be exposed client-side in the JavaScript code. To collect logs from web browsers, a [client token][2] must be used. See the [client token documentation][2] for more details.
 
-**Datadog browser log library**: Configure the library through [NPM](#npm) or use the [CDN async](#cdn-async) or [CDN sync](#cdn-sync) code snippets in the head tag.
+**Datadog browser log SDK**: Configure the SDK through [NPM](#npm) or use the [CDN async](#cdn-async) or [CDN sync](#cdn-sync) code snippets in the head tag.
```
Comment (sarina-dd) [inline packages/logs/README.md:18]:
```suggestion
**Datadog browser logs SDK**: Configure the SDK through [NPM](#npm) or use the [CDN async](#cdn-async) or [CDN sync](#cdn-sync) code snippets in the head tag.
```

File: packages/logs/README.md [hunk 2]
```diff
@@ -13,19 +13,19 @@ With the `datadog-logs` library, you can send logs directly to Datadog from JS c
...
+| Installation method        | Use case                                                                                                                                                                                                                                                                                                                                                            |
+| -------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| npm (node package manager) | This method is recommended for modern web applications. The browser logs SDK gets packaged with the rest of your front-end javascript code. It has no impact on page load performance. However, the SDK might miss errors, resources and user actions triggered before the SDK is initialized. **Note:** we recommend to use matching version with RUM SDK if used. |
```
Comment (sarina-dd) [inline packages/logs/README.md:26]:
```suggestion
| npm (node package manager) | This method is recommended for modern web applications. The browser logs SDK gets packaged with the rest of your front-end javascript code. It has no impact on page load performance. However, the SDK might miss errors, resources and user actions triggered before the SDK is initialized. **Note:** It is recommend to use a matching version with RUM SDK if used. |
```

File: packages/rum/README.md [hunk 3]
```diff
@@ -18,11 +18,11 @@ To set up Datadog RUM browser monitoring:
...
+| Installation method        | Use case                                                                                                                                                                                                                                                                                                                                                    |
+| -------------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| npm (node package manager) | This method is recommended for modern web applications. The RUM SDK gets packaged with the rest of your front-end javascript code. It has no impact on page load performance. However, the SDK might miss errors, resources and user actions triggered before the SDK is initialized. **Note:** we recommend to use matching version with logs SDK if used. |
```
Comment (sarina-dd) [inline packages/rum/README.md:23]:
```suggestion
| npm (node package manager) | This method is recommended for modern web applications. The RUM SDK gets packaged with the rest of your front-end javascript code. It has no impact on page load performance. However, the SDK might miss errors, resources and user actions triggered before the SDK is initialized. **Note:** It is recommend to use a matching version with logs SDK if used. |
```

**Review by sarina-dd — COMMENTED** [review]
Small edits

**Pull request merged**